mods/ts/sequences/infantry.yaml

```diff
@@ -140,6 +140,7 @@ oxanna:
 slav:
 	Inherits: ^BasicInfantry
 	Inherits@attack: ^BasicInfantryAttack
+	icon: xxicon
```

Comment:
Indeed, that's why I suggested this as my first choice option.  That isn't an excuse to hide and ignore whatever sequence merging bug was causing the lint failures, though.